packages/shopware-6-client/src/services/contextService.ts

```diff
@@ -159,10 +159,14 @@ export async function getAvailableSalutations(
  * @beta
  */
 export async function getAvailablePaymentMethods(
-  contextInstance: ShopwareApiInstance = defaultInstance
+  contextInstance: ShopwareApiInstance = defaultInstance,
+  params: any = {}
 ): Promise<PaymentMethod[]> {
   const resp = await contextInstance.invoke.get(
-    getContextPaymentMethodEndpoint()
+    getContextPaymentMethodEndpoint(),
+    {
+      params, // passing it as a last parameter won't cause a BC - to consider
```

Comment:
is this comment necessary? There shouldn't be any BC here :)

Reply:
yes, removed